.github/workflows/codeql.yml

```diff
@@ -0,0 +1,46 @@
+name: "Code Scanning - Action"
```

Comment:
Why is this file added to the PR?

Reply:
not sure how it showed up, or if it showed after a merge from main.  I would prefer to remove it.